.envrc.example

```diff
@@ -2,3 +2,4 @@ export POSTGRES_USER=some-user
 export POSTGRES_PASS=some-password
 export POSTGRES_DB=some-db-name
 export POSTGRES_CONNECTION_NAME=some-project:some-region:some-instance
+export POSTGRES_IAM_USER=some-iam-user
```

Comment:
nit: but I think we should be consistent with these. In the proxy we use one and java/python the other. So:
* `POSTGRES_IAM_USER` or
* `POSTGRES_USER_IAM`

Reply:
Good catch. I'll follow the pattern we already have with `POSTGRES_USER_IAM`.